### DISCUSSION TITLE
Frsky R9m- elrs tx flashing

### DISCUSSION THREAD

**Discussion by eksperymento** [OpenI6X/opentx#289] [Q&A]
Is there a way to flash a Frsky R9m tx (or any other natively non-ELRS hardware) with ELRS using the Flysky fsi6x with OpenI6x?
I've got a R9m tx on stock BL, and I'm looking for a solution to flash it, so I can do it without having to disassemble the module.
If not with the radio, maybe I can use a ftdi adapter (TX,RX)?

**Comment by ajjjjjjjj** [comment]
There is no way to do it using Flysky i6X. Look for tutorials over internet.
